include/bmi/AbstractCLibBmiAdapter.hpp

```diff
@@ -27,31 +26,17 @@ namespace models {
              */
             AbstractCLibBmiAdapter(const std::string &type_name, std::string library_file_path, std::string bmi_init_config,
```

Comment:
Formatting is a bit odd in the diff, but I didn't want to mix all these changes with formatting changes as well.